Dockerfile

```diff
@@ -0,0 +1,31 @@
+#base images
+FROM ubuntu:18.04
+FROM python:3.8
+
+#copy env file
+WORKDIR /src
+
+#install dependencies    
+RUN pip install aif360 && \
+    pip install 'aif360[LFR,OptimPreproc]' && \
+    pip install 'aif360[all]' && \
+    pip install aif360
+
+#clone repo
+RUN git clone https://github.com/Trusted-AI/AIF360.git
+
+#copy dataset
+COPY aif360/data/raw/meps/h181.csv /src/AIF360/aif360/data/raw/meps/h181.csv
+COPY aif360/data/raw/meps/h192.csv /src/AIF360/aif360/data/raw/meps/h192.csv
+
+
+#change work directory
+WORKDIR /src/AIF360
+
+#compile notebooks to run in jupyter lab
+RUN pip install --editable '.[all]' && \
```

Comment:
you are not debugging any code right, so you don't need the -e or --editable flag

Reply:
@atinsood readme instructions suggest using --editable for compilation, if the compilation fails I believe it should be verbose enough to help users figure out the issue.